### PR TITLE
upgraded Elasticesearch curator

### PR DESCRIPTION
**Pull request by xh3b4sd**
This PR goes towards https://github.com/giantswarm/giantswarm/issues/725. 

Running the curator on Leaseweb gives me this now. Since we throw away all the data in Elasticsearch and the Curator is only removing data which is 14 days old, there is nothing it is doing right now.

```
Sep 06 12:24:18 0b03b1b46b2f79d2 docker[3608]: 2016-09-06 12:24:18,968 INFO      Action #1: delete_indices
Sep 06 12:24:20 0b03b1b46b2f79d2 docker[3608]: 2016-09-06 12:24:20,176 INFO      Skipping action "delete_indices" due to empty list: <class 'curator.exceptions.NoIndices'>
Sep 06 12:24:20 0b03b1b46b2f79d2 docker[3608]: 2016-09-06 12:24:20,176 INFO      Action #1: completed
Sep 06 12:24:20 0b03b1b46b2f79d2 docker[3608]: 2016-09-06 12:24:20,176 INFO      Job completed.
```

See also https://github.com/giantswarm/releaseit/pull/551

RFR @giantswarm/team-positive

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
Do we want to wait till it starts deleting indexes?

File: config/curator.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+# Remember, leave a key empty if there is no value. None will be a string, not
+# a Python "NoneType"
+client:
+  hosts: [ '127.0.0.1' ]
+  port: 9200
+  url_prefix:
+  use_ssl: False
+  certificate:
```
Comment (hectorj2f) [inline config/curator.yml:9]: Shouldn't we omit the empty values ? I believe they add them in the official repo for the people to learn the available parameters

Comment (xh3b4sd) [reply config/curator.yml:9]: No strong feelings, but I would like to keep the defaults as much as possible.

**Comment by xh3b4sd** [comment]
@JosephSalisbury what do you mean by waiting for? It is currently deployed on Leaseweb. This is the last piece of the puzzle to finish off the Elasticsearch upgrade.

**Comment by JosephSalisbury** [comment]
> the Curator is only removing data which is 14 days old, there is nothing it is doing right now.

I was under the impression that it hasn't actually deleted anything yet. Possibly me being too conservative. We can always drop a ticket to check in a week or two.

**Comment by xh3b4sd** [comment]
See https://github.com/giantswarm/giantswarm/issues/761

**Comment by JosephSalisbury** [comment]
Cool, LGTM

**Comment by xh3b4sd** [comment]
I checked out the dry run mode and got this. Looks good to me though. 

```
Sep 07 09:23:13 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:13,848 INFO      Action #1: delete_indices
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,061 INFO      DRY-RUN MODE.  No changes will be made.
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      (CLOSED) indices may be shown that may not be acted on by action "delete_indices".
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      DRY-RUN: delete_indices: logstash-2016-09-01 with arguments: {}
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      DRY-RUN: delete_indices: logstash-2016-09-02 with arguments: {}
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      DRY-RUN: delete_indices: logstash-2016-09-03 with arguments: {}
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      DRY-RUN: delete_indices: logstash-2016-09-04 with arguments: {}
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      Action #1: completed
Sep 07 09:23:14 0b03b1b46b2f79d2 docker[22789]: 2016-09-07 09:23:14,062 INFO      Job completed.
```

**Pull request merged**